File: pojos/abono.php
<?php

class Abono
{
	public $idAbono;
	public $idTratamientoDental;
	public $fechaAbono;
	public $monto;

	function initClass($idAbono,$idTratamientoDental,$fechaAbono,$monto)
	{
		$this->idAbono = $idAbono;
		$this->idTratamientoDental = $idTratamientoDental;
		$this->fechaAbono = $fechaAbono;
		$this->monto = $monto;
	}
}
?>

File: SFH-Modulo-Escritorio Ver Anteriores/SFH-Modulo-Escritorio  ver 1.2/ObjectsBeans/Abono.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace ObjectsBeans
{
    public class Abono
    {
        #region Campos 
        private int idAbono;
        private int idTratamientoDental;
        private DateTime fechaAbono;
        private int monto;
        #endregion 

        #region Constructor
        public Abono() { 
        
        }
        #endregion

        #region Propiedad
        
        public int IdAbono
        {
            get { return idAbono; }
            set { idAbono = value; }
        }
        public int IdTratamientoDental
        {
            get { return idTratamientoDental; }
            set { idTratamientoDental = value; }
        }
        public DateTime FechaAbono
        {
            get { return fechaAbono; }
            set { fechaAbono = value; }
        }
        public int Monto
        {
            get { return monto; }
            set { monto = value; }
        }
        #endregion
    }
}
